addons/web/static/src/search/control_panel/control_panel.scss

```diff
@@ -37,7 +37,7 @@
         > .btn-group {
             margin-bottom: map-get($spacers, 2);
             // Before owl, the margin was handled by a real space!
-            margin-right: map-get($spacers, 1);
```

Comment:
when used outside the control panel --> don't agree with that: the GroupByMenu of a subview graph is also in a control panel

Reply:
Some border**s**